docs/getting_started.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started
+
+Cortex can be runs as a single binary or as multiple independent microservices.
```

Comment:
Can you be more precie about this? You could clarify the advantages/disadvantages of each approach. Right now I could only guess, that multiple independent microservices offer the advantage to be independently scalable.

Also what 3rd party dependencies do each of these approaches have? Does one always need memcache, consul & a timeseries DB such as Bigtable?

Reply:
Also what do you think about adding a link to https://github.com/cortexproject/cortex/blob/master/docs/architecture.md ? This gave me a good high level overview about how cortex works and thus helped me understanding it.

Reply:
Done!  Thanks for the feedback.